# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/hyprcursor/hotfix-system-dir.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- hyprcursor-0.1.9/libhyprcursor/hyprcursor.cpp.orig	2024-07-03 16:38:26.565501218 +0200
+++ hyprcursor-0.1.9/libhyprcursor/hyprcursor.cpp	2024-07-03 16:38:45.621499382 +0200
@@ -16,7 +16,7 @@
 using namespace Hyprcursor;
 
 // directories for lookup
-constexpr const std::array<const char*, 1> systemThemeDirs = {"/usr/share/icons"};
+constexpr const std::array<const char*, 1> systemThemeDirs = {"/usr/X11/share/icons", "/usr/share/icons"};
 constexpr const std::array<const char*, 2> userThemeDirs   = {"/.local/share/icons", "/.icons"};
 
 //
